testing/ef_tests/tests/tests.rs

```diff
@@ -101,6 +101,8 @@ fn sanity_blocks() {
 }
 
 #[test]
+// One of these EF tests causes a sync committee calculation, which requires pubkey aggregation
+#[cfg(not(feature = "fake_crypto"))]
```

Comment:
I would prefer to put this on the `SanitySlotsHandler` to keep consistency with the other handlers and cases which use `is_enabled_for_fork`:

```rust
fn is_enabled_for_fork(&self, _fork_name: ForkName) -> bool {
    // NOTE: v1.1.0-beta.4 doesn't mark the historical blocks test as requiring real crypto, so
    // only run these tests with real crypto for now.
    cfg!(not(feature = "fake_crypto"))
}
```